unmarshal.go

```diff
@@ -601,9 +601,9 @@ func unmarshalConnectionInformation(value string) (*ConnectionInformation, error
 		return nil, fmt.Errorf("sdp: invalid value `%v`", fields[1])
 	}
 
-	var connAddr *Address
+	connAddr := new(Address)
```

Comment:
This seems adding whitespace at the end of `c=` line like `v=0\no=0 0 0 IN IP4 0\ns=0\nc=IN IP4 \nt=0 0\n`.
It would be nice to have a roundtrip test.

Reply:
This is not due to this change; it is because of how ConnectionInformation.String is generated. I've added a commit to this PR to fix it, though.